crates/bevy_input/src/touch.rs

```diff
@@ -142,51 +189,76 @@ impl From<&TouchInput> for Touch {
     }
 }
 
+/// A collection of [`Touch`]es.
+///
+/// ## Usage
+///
+/// It is used to create a `bevy` resource that stores the data of the touches on a touchscreen
+/// and can be accessed inside of a system.
+///
+/// ## Updating
+///
+/// The resource is updated inside of the [`touch_screen_input_system`](crate::touch::touch_screen_input_system).
 #[derive(Debug, Clone, Default)]
 pub struct Touches {
+    /// A collection of every button that is currently being pressed.
```

Comment:
Is button the right terminology here? I haven't seen buttons be mentioned anywhere else yet